source/Octopus.Manager.Tentacle.Tests/WizardFixtures/SetupTentacleWizardFixture.cs

```diff
@@ -96,5 +96,23 @@ await telemetryService
                         t.EventProperties.ContainsKey("Machine Type") &&
                         t.EventProperties["Machine Type"] == MachineType.DeploymentTarget.ToString()));
         }
+
+        [Test]
+        public async Task WhenSettingUpAListeningTentacle_TelemetryEventShouldNotBeSent()
+        {
+            var telemetryService = new TelemetryServiceBuilder().Build();
+
+            var model = new SetupTentacleWizardModelBuilder()
+                .WithTelemetryService(telemetryService)
+                .Build();
+
+            model.CommunicationStyle = CommunicationStyle.TentaclePassive;
+
+            _ = await model.ReviewAndRunScriptTabViewModel.GenerateAndExecuteScript();
+
+            await telemetryService
+                .Received(0)
```

Comment:
Would `DidNotReceive()` be clearer here?

https://nsubstitute.github.io/help/received-calls/#check-a-call-was-not-received

Reply:
It would 🙇🏻